### DISCUSSION TITLE
VideoWriter node is saving video with 0 bytes

### DISCUSSION THREAD

**Discussion by Mel-Gonzalez2k** [bonsai-rx/bonsai#1475] [Q&A]
Hello! 

I'm using Bonsai + Deeplabcut to track and record a video from a Basler camera on a Windows 10 machine. I'm troubleshooting with a simplified workflow of camera capture and video writer, the frames are being acquired and I can see the video displayed but the saved video is empty/has 0KB size. I was able to record and save videos with this workflow a few days ago, but this issue recently appeared and I'm not sure what is wrong. I also have no issues with storage on my computer, and I've tried saving the video onto different drivers and I've tried different video codecs.

Thank you for your help! 

![camera_video](https://github.com/bonsai-rx/bonsai/assets/115488381/2801056b-2def-4d9a-950e-7a9f666c5c73)

**Comment by ckosse** [comment]
I have exactly the same problem- Did you manage to solve it?

**Reply by ckosse** [reply]
I did a clean install of Bonsai and it works again for me! I hope this helps  :)
